swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        
+        if id is None:
+            unique_id = uuid.uuid4()
+            self.id = unique_id.int
+        else:
+            self.id = id         
+
+        self.condition = condition      
+
+    def get_category(self):
+        return "Item"   
```

Comment:
👍 It may seem silly, but returning the string literal here sets us up for overriding the `get_category` function in later child classes. There are other ways we could investigate doing this, but we would want to be careful that our category value, which is a user-facing piece of business logic, doesn't become intertwined with the class name, since class names are often named with much less user-friendly names.